docs/features/configuration.md

```diff
@@ -72,3 +72,15 @@ but does not allow starting privileged containers, you can turn off the Ryuk con
 
 > **pull.pause.timeout = 30**
 > By default Testcontainers will abort the pull of an image if the pull appears stalled (no data transferred) for longer than this duration (in seconds).
+
+## Customizing Docker host detection
+
+Testcontainers will attempt to detect the Docker environment and configure everything.
```

Comment:
If we're saying this, should we (for completeness) mention DOCKER_HOST?

Reply:
Added 👍